circleapi.go

```diff
@@ -0,0 +1,129 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	circleci "github.com/maplebed/go-circleci"
```

Comment:
recommend referencing using the normal GH repo, and use `go.mod`'s `replace` directive to point it at your fork.  that way when we get off the fork the change is just in `go.mod`/`go.sum`.

Reply:
👍 did that.